### ISSUE TITLE
Dealing extremely low melee damage

### ISSUE THREAD

**Issue by alex-p-b** [CleverRaven/Cataclysm-DDA#52868]
### Describe the bug

My character deals very low melee damage for seemingly no reason.
Things I've checked:
-Making sure that I'm not ridiculously encumbered (encumbrance is below 20 in all limbs)
-Making sure that I'm not forcing unarmed strikes
-Making sure that the weapon I'm using, a broadsword, is a real broadsword and not a fake. I also tried using a mace with the same result.
-Making sure that my broadsword and mace are in adequate condition
-I've got 5 melee, 5 cutting, and 5 bashing skill
-I've got plenty of stamina left
-Making sure that I'm not wielding an unarmed weapon accidentally
-All settings are default, no mods enabled other than the ones enabled by default

### Steps To Reproduce

[auto_after_end28.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/7557497/auto_after_end28.zip)
I'm not really sure how to reproduce this bug with a fresh character, but attached is my save file. Simply try to whack something in melee and you'll see that your damage is far lower than would be normal. 

### Expected behavior

I expect to do around ~50 damage with a broadsword to a basic zombie. Instead, I do around 5.

### Versions and configuration

OS: Windows 10
Game version: b1c74e2 (tiles)
Graphics version: Tiles
Ingame language: English
Mods loaded: dda, no_npc_food, no_fungal_growth, package_bionic_professions

**Comment by kholat** [comment]
Are you very wounded? I noticed when my arms were very low HP i did like 5 damage to a child with a steel spear.

**Comment by Venera3** [comment]
It will probably be the combination of limb hp and encumbrance tipping you over into "broken arms mode". The relevant code lives in `melee.cpp:680`:
```
        // if you have two broken arms you aren't doing any martial arts
        // and your hits are not going to hurt very much
        if( blocking_score( body_part_type::type::arm ) < 1.0f ) {
            technique_id = tec_none;
            d.mult_damage( 0.1 );
        }
```
Each of your arms contribute 1 blocking score, which gets reduced linearly by their encumbrance and health (beginning at 75% HP). I'm not entirely sure this check makes any sense now that we have scores to potentially affect damage, but the easy fix would be to reduce the threshold to 0.6 or so. 

It would also be helpful if there was some sort of player-facing feedback on what's happening, but that assumes that it should be happening.
